### ISSUE TITLE
HTML Elements added at bottom of <body> and not adjacent to <canvas> in the DOM

### ISSUE THREAD

**Issue by TobyKLight** [cables-gl/cables_docs#806]
https://github.com/user-attachments/assets/e4a9d29c-f2f4-43cc-9dce-e65979c8db2c

**Describe the bug**
HTML elements added by a cables.gl patch are added to the bottom of the DOM instead of as children of the Canvas. 

This causes two issues with accessibility 
* Problems with screen readers as they read content in the order it appears in the DOM. If the cables canvas is visually in the middle of the page but its HTML content is at the end of the body then the screen reader will read it out of order.
* There are a bunch of accessibility patterns that rely on the html element containing interactive content to have appropriate tags. If the content is not actually a child of the canvas this will cause problems. 

**How To Reproduce**
I published an embedded modified version of the Div example patch to check this in the wild 
https://www.tobyk.com.au/cablestest/RegDiv2/

Check the inspector with chrome and notice the interactive DIV is at the bottom of the body after the lower fixed html content on the page.

**Platform**
Chrome Version 126.0.6478.127 (Official Build) (64-bit) on win 11

**Comment by pandrr** [comment]
you could also use queryselector and elementChilds to move the html outside of the whole cables elements.

**Comment by TobyKLight** [comment]
yes that's exactly what I'm doing, requires a bit of manual HTML work to create the elements but it works.
